src/components/schemas/report_approval.yaml

```diff
@@ -16,3 +16,9 @@ report_approval_out_embed:
         - APPROVAL_DONE
       description: |
         State of this report_approval
+    rank:
```

Comment:
Did we decide on the "rank" somewhere? IMO, since this is a public facing API it should be something like "approver_order"/"order"